themes/bootstrap3/templates/footer.phtml

```diff
@@ -23,6 +23,9 @@
         <li><a href="<?=$this->url('help-home')?>?topic=search&amp;_=<?=time() ?>" data-lightbox class="help-link"><?=$this->transEsc('Search Tips')?></a></li>
         <li><a href="<?=$this->url('content-page', ['page' => 'asklibrary']) ?>"><?=$this->transEsc('Ask a Librarian')?></a></li>
         <li><a href="<?=$this->url('content-page', ['page' => 'faq']) ?>"><?=$this->transEsc('FAQs')?></a></li>
+        <?php if ($this->config()->get('Overdrive')->Overdrive->ODHelpLink ?? false):?>
```

Comment:
This is another area where it would be nice to come up with more generic language instead of having Overdrive-specific logic in a widely-used template. For example, to display the "Powered by Summon" message required on Summon-related screens, we set `$this->layout()->poweredBy` within the appropriate controller, and there's a placeholder in the layout which makes use of that value. Could we do something similar here?

Reply:
I like the idea of an "extraHelpLinks" array that different components could dynamically add a link to. The footer would add those links to the "help" section of the footer.  

If a user wants to display the overdrive help link, I think it makes sens to display it in the footer of every page, not just the "my overdrive content" page.  What would be a good way to do that?

Reply:
One possible approach could be to build a HelpLinks view helper, which could have addLink() and renderLinks() methods. Then various components could push links into the helper through addLink(), and it could display all of the collected values through a call to renderLinks() in the footer. Perhaps that's more engineering than is worth the effort for the simple need here, but it's the first solution that comes to mind. I can help build a prototype if that would be useful.

Reply:
Where would the addLink() method get called from in my case?

Reply:
It could theoretically be called from almost anywhere. If you wanted to globally inject a string, you could call it as part of the helper's factory. If you wanted to inject it in a context-sensitive way, you could call it from templates or controllers where the link is specifically needed.

Reply:
If you think it's useful and want to set up the view helper, then I'll try to use it.  If not, I can just take the link out.  It doesn't matter to me that much.

Reply:
If you think it's worth the effort, I'm happy to help with the view helper idea... but if you're not that invested in the issue, it's obviously simpler to just remove it for now. We can always revisit the idea later as a separate project.

Reply:
Demian I finally got around to removing the link in the footer.  I believe that I'm caught up on the changes you requested back in March.  Let me know what's next.